.github/workflows/cd.yml

```diff
@@ -0,0 +1,69 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  benchmark_tests:
+    name: benchmark tests
+    runs-on: ubuntu-latest
+    env:
+      SOURCE_PROJECT_KEY: java-sync-source
+      SOURCE_CLIENT_ID: ${{ secrets.SOURCE_CLIENT_ID }}
+      SOURCE_CLIENT_SECRET: ${{ secrets.SOURCE_CLIENT_SECRET }}
+      TARGET_PROJECT_KEY: java-sync-target
+      TARGET_CLIENT_ID: ${{ secrets.TARGET_CLIENT_ID }}
+      TARGET_CLIENT_SECRET: ${{ secrets.TARGET_CLIENT_SECRET }}
+      CI_BUILD_DIR: '/home/runner/work/commercetools-sync-java/commercetools-sync-java/'
+      GRGIT_USER: ${{ secrets.GRGIT_USER }}
+      GITHUB_ACTION_COMMIT: ${{github.sha}}
```

Comment:
How this github.sha is filled ? Did you test it ?

Reply:
yes ->
![Bildschirmfoto 2021-02-03 um 15 53 36](https://user-images.githubusercontent.com/1541309/106764748-6003de00-6638-11eb-97fd-aa1dde0a8ff0.png)

Reply:
So it a default variable on Github actions, good to know 👍🏽